laboratory-service/src/main/java/com/raf/si/laboratoryservice/service/impl/LabExamServiceImpl.java

```diff
@@ -0,0 +1,92 @@
+package com.raf.si.laboratoryservice.service.impl;
+
+import com.raf.si.laboratoryservice.dto.request.CreateLabExamRequest;
+import com.raf.si.laboratoryservice.dto.request.UpdateLabExamStatusRequest;
+import com.raf.si.laboratoryservice.dto.response.LabExamResponse;
+import com.raf.si.laboratoryservice.exception.NotFoundException;
+import com.raf.si.laboratoryservice.mapper.LabExamMapper;
+import com.raf.si.laboratoryservice.model.Referral;
+import com.raf.si.laboratoryservice.model.ScheduledLabExam;
+import com.raf.si.laboratoryservice.model.enums.scheduledlabexam.ExamStatus;
+import com.raf.si.laboratoryservice.repository.ScheduledLabExamRepository;
+import com.raf.si.laboratoryservice.service.LabExamService;
+import com.raf.si.laboratoryservice.utils.TokenPayloadUtil;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Slf4j
+@Service
+public class LabExamServiceImpl implements LabExamService {
+    private final ScheduledLabExamRepository scheduledLabExamRepository;
+
+    private final LabExamMapper labExamMapper;
+
+    public LabExamServiceImpl(ScheduledLabExamRepository scheduledLabExamRepository, LabExamMapper labExamMapper) {
+        this.scheduledLabExamRepository = scheduledLabExamRepository;
+        this.labExamMapper = labExamMapper;
+    }
+
+
+    @Override
+    public LabExamResponse createExamination(CreateLabExamRequest createLabExamRequest) {
+        UUID lbzFromToken = TokenPayloadUtil.getTokenPayload().getLbz();
+        UUID pboFromToken = TokenPayloadUtil.getTokenPayload().getPbo();
+        ScheduledLabExam scheduledLabExam = scheduledLabExamRepository.save(labExamMapper.requestToModel(createLabExamRequest, lbzFromToken, pboFromToken));
+        return labExamMapper.modelToResponse(scheduledLabExam);
+    }
+
+    @Override
+    public Optional<Long> getScheduledExamCount(Timestamp date) {
+        LocalDateTime localDateTime = date.toLocalDateTime().plusDays(1).withHour(0).withMinute(0).withSecond(0);
+        Timestamp endDate = Timestamp.valueOf(localDateTime);
```

Comment:
Treba samo odseci i sate, minute i sekunde sa date-a, jer se u bazi date-ovi cuvaju sa satima, minutama i sekundama, pa ako je prosledjen date koji ima sate, minute ili sekunde, moze da dodje do problema. Mada, ako se stavi za @RequestParam format datuma koji ima samo godine, mesece i dane nemamo taj problem. Na to mora da se obrati paznja kada se dobija datum iz requesta, jer nisam siguran da li tu moze da se navede format koji treba da se koristim